src/main/java/core/basesyntax/NumberSupplier.java

```diff
@@ -0,0 +1,2 @@
+package core.basesyntax;public class NumberSupplier {
```

Comment:
I belive aftermpackage name should be line break as in your other classes, for example src/main/java/core/basesyntax/model/Ball.java or src/main/java/core/basesyntax/service/Lottery.java